crates/tabby/src/services/code.rs

```diff
@@ -75,16 +75,16 @@ impl CodeSearchImpl {
             })
             .await?;
 
-        let Some(git_url) = closest_match(&query.git_url, repos.iter()) else {
+        let Some(source_id) = closest_match(&query.git_url, repos.iter()) else {
```

Comment:
`closet_match` will still return `git_url`. But the source_id shall be like `config_index_to_id` ? Is it still right?

Reply:
closet match has been changed to return source id already